### PR TITLE
change password script

### PR DESCRIPTION
**Pull request by alsotang**
related to #1050

---
This change is [Reviewable](https://reviewable.io/reviews/cnpm/cnpmjs.org/1070)

### CODE REVIEW

**Comment by fengmk2** [comment]
eslint fail

image

File: bin/change_password.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// 只适用于使用 DefaultUserService 的场景，自定义用户鉴权方式的用户请忽略此文件
```
Comment (fengmk2) [inline bin/change_password.js]: 改成英文吧

**Comment by alsotang** [comment]
Done.

use english. add semicolon

**Comment by fengmk2** [comment]
+1

**Pull request merged**